.github/workflows/perf-testing-setup.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         python setup.py extension
     - name: Build and Run the Docker image
       run: |
-        docker build --build-arg PYTHON_VERSION=${{ env.PYTHON_VERSION }} --file .ci/perf_tests/dockerfiles/perf_tests.Dockerfile --tag perfimage:latest .
+        docker build --build-arg PYTHON_VERSION=${{ env.PYTHON_VERSION }} --build-arg STORAGE_CONNECTION_STRING=${{ secrets.LinuxStorageConnectionString38 }} --file .ci/perf_tests/dockerfiles/perf_tests.Dockerfile --tag perfimage:latest .
```

Comment:
Instead of this, let's do `--env AzureWebJobsStorage=${{ secrets.LinuxStorageConnectionString38 }}`